src/M6Web/Bundle/LogBridgeBundle/Config/FilterParser.php

```diff
@@ -57,169 +44,99 @@ protected function createFilter($name)
         return new Filter($name);
     }
 
-    /**
-     * isRoute
-     *
-     * @param string $name Route name
-     *
-     * @return bool
-     */
-    protected function isRoute($name)
+    protected function isRoute(string $name): bool
     {
-        return $this->router->getRouteCollection()->get($name) ? true : false;
+        return $this->router->getRouteCollection()->get($name) !== null;
     }
 
-    /**
-     * parse
-     *
-     * @param string $name   name
-     * @param array  $config configuration
-     *
-     * @throws ParseException
-     *
-     * @internal param array $filterConfig
-     *
-     * @return Filter
-     */
-    public function parse($name, array $config)
+    public function parse(string $name, array $config): Filter
     {
-        $filter = $this->createFilter($name);
-
-        if (!array_key_exists('route', $config) || !array_key_exists('method', $config) || !array_key_exists('status', $config)) {
+        if (
+            !array_key_exists('route', $config) ||
+            !array_key_exists('method', $config) ||
+            !array_key_exists('status', $config)
+        ) {
             throw new ParseException(sprintf('Undefined "route", "method" or "status" parameter from filter "%s"', $name));
         }
 
         if (!array_key_exists('level', $config)) {
             $config['level'] = self::DEFAULT_LEVEL;
         }
 
+        $filter = $this->createFilter($name);
+
         $this->parseRoute($filter, $config['route']);
         $this->parseMethod($filter, $config['method']);
         $this->parseStatus($filter, $config['status']);
         $this->parseLevel($filter, $config['level']);
 
-        $filter->setOptions(isset($config['options']) ? $config['options'] : []);
-
-        return $filter;
+        return $filter->setOptions($config['options'] ?? []);
     }
 
-    /**
-     * parseRoute
-     *
-     * @param Filter $filter Filter
-     * @param mixed  $route  Route parameter value
-     *
-     * @throws ParseException
-     */
-    protected function parseRoute(Filter $filter, $route)
+    protected function parseRoute(Filter $filter, ?string $route): void
```

Comment:
As we throw an exception if $route is null, why we allow here ?
Non-blocking suggestion:
```suggestion
    protected function parseRoute(Filter $filter, string $route): void
```

Reply:
I'd say because the code is messy... but again we should not break too much things in this PR, so I would even suggest to revert to:

```suggestion
    protected function parseRoute(Filter $filter, mixed $route): void
```

for now, to avoid throwing a `TypeError` instead of a `ParseException` when the function is misused... 

=> of course this would be a good idea to have a stricter typing (to have a code more maintainable in our side and easier to understand for its users), but maybe in another PR where we focus on these breaking changes?

Reply:
You're right, I'll change this and the others, there's no point in allowing null & throwing an exception if it is. 
I'll push a new commit with your suggestion and all it implies